gpu_direct_rdma_access.h

```diff
@@ -75,6 +75,11 @@ struct rdma_open_dev_attr {
 struct rdma_device *rdma_open_device_client(struct sockaddr *addr);
 struct rdma_device *rdma_open_device_server(struct sockaddr *addr);
 
+/*
+ * Reset device state
```

Comment:
make add more to the comment. 
"..reset a error'ed rdma_device back to an operations state"
so users can understand this should not be called immediately after creating the server device.

Reply:
done